telethon/telegram_bare_client.py

```diff
@@ -69,6 +69,7 @@ class TelegramBareClient:
 
     def __init__(self, session, api_id, api_hash,
                  connection_mode=ConnectionMode.TCP_FULL,
+                 use_v6=False,
```

Comment:
`v6` sounds like "version six". Version six of what? Maybe something more explicit like `use_ipv6` would make more sense. I'll change this.

Reply:
Yea, I also think that it will be better.